mypy/test/testargs.py

```diff
@@ -5,13 +5,15 @@
 object it creates.
 """
 
-from mypy.myunit import Suite, assert_equal
-from mypy.options import Options, BuildType
+from mypy.test.helpers import Suite
+from mypy.options import Options
 from mypy.main import process_options
 
 
 class ArgSuite(Suite):
     def test_coherence(self) -> None:
         options = Options()
         _, parsed_options = process_options([], require_targets=False)
-        assert_equal(options, parsed_options)
+        assert options.__dict__.keys() == parsed_options.__dict__.keys()
```

Comment:
Could you explain why you chose this method of checking equality? `assert_equal` is essentially `assert not a != b`, I believe this could be a bit different, and is not as straightforward.

Reply:
I added these checks in order to find out why it failed. Since it was helpful for me, I thought it might be better not to switch back.